src/scripts/run-production-migration.js

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env node
+
+/**
+ * Production Migration Runner for Mentoring Service
+ * Handles 30+ lakh records with Citus distribution
+ */
+
+require('dotenv').config({ path: '../.env' })
+const MentoringDataMigrator = require('./mentoring-data-migration')
+const readline = require('readline')
+
+console.log('🎯 Production Mentoring Service Data Migration')
+console.log('==============================================')
+
+// Configuration check
+console.log('\n📋 Environment Configuration:')
+console.log(`   Database URL: ${process.env.DEV_DATABASE_URL ? '✅ Set' : '❌ Missing'}`)
+console.log(`   Default Tenant: ${process.env.DEFAULT_ORGANISATION_CODE || 'DEFAULT_TENANT'}`)
+console.log(`   Default Org Code: ${process.env.DEFAULT_ORG_CODE || 'DEFAULT_ORG'}`)
+console.log(`   Default Org ID: ${process.env.DEFAULT_ORG_ID || '1'}`)
+
+// Check CSV files
+const fs = require('fs')
+const path = require('path')
+
+console.log('\n📁 CSV Files Status:')
+const csvFiles = ['data_codes.csv']
+let allFilesExist = true
+
+csvFiles.forEach((file) => {
+	const filePath = path.join(__dirname, '../data', file)
+	const exists = fs.existsSync(filePath)
+	console.log(`   ${file}: ${exists ? '✅ Found' : '❌ Missing'}`)
+	if (!exists) allFilesExist = false
+
+	if (exists) {
+		const stats = fs.statSync(filePath)
+		const sizeKB = Math.round(stats.size / 1024)
+		console.log(`     Size: ${sizeKB}KB, Modified: ${stats.mtime.toISOString().split('T')[0]}`)
+	}
+})
+
+// Migration plan
+console.log('\n📋 Migration Plan:')
+console.log('   Phase 1: Tables with organization_id using GROUP BY strategy')
```

Comment:
@sumanvpacewisdom improve the language, not able to understand, also its step wise